lighthouse-core/test/lib/style-helpers-test.js

```diff
@@ -63,14 +67,17 @@ describe('style helpers', () => {
       results = StyleHelpers.filterStylesheetsByUsage(
           stylesheets, 'display', 'someunknownval');
       assert.equal(results.length, 0, 'known CSS property with unknown value not found');
+
+      results = StyleHelpers.filterStylesheetsByUsage(
+          stylesheets, ['display'], 'someunknownval');
+      assert.equal(results.length, 0, 'known CSS property in array with unknown value not found');
     });
   });
 
   describe('getFormattedStyleRule()', function() {
     it('formats output correctly', () => {
       const results = StyleHelpers.filterStylesheetsByUsage(
-          stylesheets, 'display', 'box');
```

Comment:
can you keep this string param test and add your `['display'], ['box']` param permutation?